### ISSUE TITLE
[BUG] downtime does not accept vars

### ISSUE THREAD

**Issue by AngelNev** [tribe29/ansible-collection-tribe29.checkmk#243]
Verify first that your issue is not already reported [here](https://github.com/tribe29/ansible-collection-tribe29.checkmk/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc).
Where possible also test if the latest release and main branch are affected too.
Complete all sections as described!

**Describe the bug**
Using a variable from Jenkins and Ansible gives a error, saying "unsupported type for timedelta minutes component: str"
Using a fix int as in the example works. I think, the actual Pyhton script does not accept strings or should transform/assure it is an integer.

**Component Name**
`downtime`

**Ansible Version**
```console
ansible 2.9.27
```
**Checkmk Version**
```console
2.1.0p16
```
**Collection Version**
```console
0.16.1
```
**Environment**
RHEL8

**To Reproduce**
```console
- set_fact:
     START_AFTER_MINUTES: '{{ START_AFTER_MINUTES | int }}'

- debug:
    msg: "The 'START_AFTER_MINUTES' variable is NEW a {{ START_AFTER_MINUTES | type_debug }}"

- name: "Schedule host downtime."
      downtime:
        server_url: "{{ server_url }}"
        site: "{{ site }}"
        automation_user: "{{ automation_user }}"
        automation_secret: "{{ automation_secret }}"
        comment: "{{ COMMENT }}"
        host_name: "{{ SERVERNAME }}"
        start_after:
          minutes: "{{ START_AFTER_MINUTES }}"
        end_after:
          hours: 2
        state: "present"
        validate_certs: false
      delegate_to: localhost
```
**Expected behavior**
The host goes into maintenance.

**Actual behavior**
unsupported type for timedelta minutes component: str

**Comment by robin-tribe29** [comment]
Hi @AngelNev and thanks for your report.
Can you provide the exact error message, that you receive? Thanks!

**Comment by AngelNev** [comment]
Hi Robin, sure and sorry, yesterday I was in a hurry:

TASK [Schedule host downtime.] *************************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: TypeError: unsupported type for timedelta minutes component: str
fatal: [host.fqdn -> localhost]: FAILED! => {"changed": false, "module_stderr": "Traceback (most recent call last):\n  File \"/export/home/jenkins/.ansible/tmp/ansible-tmp-1674654859.015577-1673643-5844787178732/AnsiballZ_downtime.py\", line 102, in <module>\n    _ansiballz_main()\n  File \"/export/home/jenkins/.ansible/tmp/ansible-tmp-1674654859.015577-1673643-5844787178732/AnsiballZ_downtime.py\", line 94, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/export/home/jenkins/.ansible/tmp/ansible-tmp-1674654859.015577-1673643-5844787178732/AnsiballZ_downtime.py\", line 40, in invoke_module\n    runpy.run_module(mod_name='ansible_collections.tribe29.checkmk.plugins.modules.downtime', init_globals=None, run_name='__main__', alter_sys=True)\n  File \"/usr/lib64/python3.6/runpy.py\", line 205, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib64/python3.6/runpy.py\", line 96, in _run_module_code\n    mod_name, mod_spec, pkg_name, script_name)\n  File \"/usr/lib64/python3.6/runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_downtime_payload_jfdkypvw/ansible_downtime_payload.zip/ansible_collections/tribe29/checkmk/plugins/modules/downtime.py\", line 464, in <module>\n  File \"/tmp/ansible_downtime_payload_jfdkypvw/ansible_downtime_payload.zip/ansible_collections/tribe29/checkmk/plugins/modules/downtime.py\", line 460, in main\n  File \"/tmp/ansible_downtime_payload_jfdkypvw/ansible_downtime_payload.zip/ansible_collections/tribe29/checkmk/plugins/modules/downtime.py\", line 448, in run_module\n  File \"/tmp/ansible_downtime_payload_jfdkypvw/ansible_downtime_payload.zip/ansible_collections/tribe29/checkmk/plugins/modules/downtime.py\", line 295, in set_downtime\n  File \"/tmp/ansible_downtime_payload_jfdkypvw/ansible_downtime_payload.zip/ansible_collections/tribe29/checkmk/plugins/modules/downtime.py\", line 185, in _set_timestamps\nTypeError: unsupported type for timedelta minutes component: str\n", "module_stdout": "", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}

**Comment by robin-tribe29** [comment]
@AngelNev there is already a [fix](https://github.com/tribe29/ansible-collection-tribe29.checkmk/pull/244) ready to merge, but we are halting for a few days. We will merge within two weeks and this issue will close automatically. Keep an eye out on the next releases. :v:

**Comment by AngelNev** [comment]
That's great! :)

**Comment by robin-tribe29** [comment]
This is fixed by #251.

**Comment by AngelNev** [comment]
Just tested out and all works fine as expected - thanks a lot!
